tests/Unit/Facade.Tests/ExportRsnService/WhenGettingExportRsnsWithNumber.cs

```diff
@@ -25,7 +25,7 @@ public void SetUp()
 
             this.ExportRsnRepository.FilterBy(Arg.Any<Expression<Func<ExportRsn, bool>>>()).Returns(rsns.AsQueryable());
 
-            this.result = this.Sut.SearchRsns(123, null, "2");
+            this.result = this.Sut.SearchRsns(123, null, "2","Y");
```

Comment:
looks like your stylecop might have switched off if it's not complaining about the space here.